transaction-overview.md

```diff
@@ -293,7 +293,7 @@ mysql> SELECT * FROM test;
 
```

Comment:
@cfzjywxk Can you help address this feedback?
> 针对 [事务限制](https://docs.pingcap.com/zh/tidb/stable/transaction-overview#%E4%BA%8B%E5%8A%A1%E9%99%90%E5%88%B6) 中描述的 「由于底层存储引擎的限制，TiDB 要求单行不超过 6 MB。可以将一行的所有列根据类型转换为字节数并加和来估算单行大小。」目前已经支持参数进行调整，需要增加参数来告诉用户该如何调整，且务必需要说明，调整改参数可能带来的后果，譬如超出 raftstore message 的大小，或者 grpc send msg 的大小

Reply:
It is a much more complex topic, it's better to do the enhancement task seperately.

Reply:
ok